ros_ws/src/get_user_request.py

```diff
@@ -0,0 +1,141 @@
+import os
+import langchain
+
+from langchain.llms import OpenAI
+from langchain import PromptTemplate
+from langchain import schema
+from langchain.prompts import (ChatPromptTemplate,PromptTemplate,SystemMessagePromptTemplate, HumanMessagePromptTemplate)
+from langchain.chat_models import ChatOpenAI
+from langchain.chains import LLMChain
+from langchain.schema import HumanMessage, SystemMessage, AIMessage
+api_key=os.environ["OPENAI_API_KEY"]
+model_to_use="gpt-4-0613"
```

Comment:
GPT-4 turbo just came out and is 1/3rd the price and much faster. Might be worth exploring. I've mostly been using gpt3.5-turbo for its price and speed.

Reply:
I'll give turo a run